core/src/main/java/de/jplag/JPlagResult.java

```diff
@@ -10,7 +10,15 @@
 import de.jplag.options.SimilarityMetric;
 
 /**
- * Encapsulates the results of a comparison of a set of source code submissions.
+ * Encapsulates the results of a pairwise comparison of program structure among a set of source code submissions.
+ * <p>
+ * Provides access to:
+ * <ul>
+ * <li>Pairwise comparison results sorted by similarity</li>
+ * <li>Similarity distribution data</li>
+ * <li>Execution duration and configuration options</li>
```

Comment:
I remember a case where we said we would rather not have HTML in the documentation. I think it is helpful here, though!

Reply:
Good point! I will restructure it to omit the HTML-style syntax.